knowledge_graph/querying_raw.py

```diff
@@ -70,9 +70,19 @@ def triplets_with_relation(
     return gamla.compose_left(triplets_index.relation_index, gamla.apply(relation))
 
 
-all_display_and_trigger_triplets = gamla.juxtcat(
-    triplets_with_relation(common_relations.TRIGGER),
-    triplets_with_relation(common_relations.DISPLAY),
+def triplets_with_relations(
+    relations: Iterable[triplet.Element],
+) -> Callable[[triplets_index.TripletsWithIndex], Iterable[triplet.Triplet]]:
+    return gamla.pipe(
+        relations,
+        gamla.map(triplets_with_relation),
+        gamla.star(gamla.juxt),
```

Comment:
```suggestion
        gamla.star(gamla.juxctat),
```
(and remove the line below)